typescript/sdk/src/utils/filter.ts

```diff
@@ -0,0 +1,13 @@
+import { ChainMap } from '../types';
+
+export function filterByChains<T>(
```

Comment:
Moved here from sdk/src/utils/objects, the rest of which was moved to the utils package